content/routers/_index.md

```diff
@@ -16,3 +16,19 @@ oc patch \
    --type=merge \
    ingresscontroller/default
 ```
+
+# Moving routers to a specific (infra) node 
+
+Label the desired node with a particular label (infra)
+```
+oc label node worker1  node-role.kubernetes.io/infra=
+oc label node worker2  node-role.kubernetes.io/infra=
+oc adm taint nodes -l node-role.kubernetes.io/infra infra=reserved:NoSchedule infra=reserved:NoExecute
+```
+
+Patch the ingresscontroller to use the nodes with specific nodeselector
```

Comment:
```suggestion
Patch the `ingresscontroller` to use the nodes with specific `nodeselector` and a toleration for the previous `taint`:
```

Reply:
I am good. Thanks for adding more explanation.

Reply:
Would you mind to add those changes? Thanks!